packages/story-editor/src/components/canvas/shoppingPageAttachment/index.js

```diff
@@ -119,7 +119,7 @@ function PageAttachment({ ctaText, theme }) {
   }));
 
   return (
-    <Wrapper role="presentation">
+    <Wrapper role="presentation" tabIndex={-1}>
```

Comment:
Notice the difference when tabbing - the first part is without a product and works as expected, then the issue occurs when there is a product on the Canvas.
`Shop Now` is causing the issues, I think it doesn't need to be focusable at all and this change fixes the issue but PTAL @barklund.

https://user-images.githubusercontent.com/21036224/199951874-34bca72b-7684-4a7c-a729-17fa272a060e.mp4

Reply:
Looks like there is a bigger issue with focusing via tab.

https://user-images.githubusercontent.com/21036224/199968876-9f577ed3-321e-45d5-836b-7b2d676c48ee.mp4

Reply:
Hmm, this sounds mysterious and definitely like a bug. This was originally implemented by Pea Pod, so I don't know all the details of the canvas tabbing container, but I'll look into it.

Reply:
It works if you get to the canvas container going backwards (so start focusing the story title input, then tab backwards, and you're good. Tab past the canvas container, and start tabbing forward, and it breaks!)